tools/codegen/gen_backend_stubs.py

```diff
@@ -133,13 +135,14 @@ def create_backend_index(
 autograd key. They cannot be mix and matched. If this is something you need, feel free to create an issue! \
 {forward_kernels[0].kernel} is listed under "supported", but {backward_kernels[0].kernel} is listed under "autograd".'
 
-    return ParsedExternalYaml(backend_key, autograd_key, cpp_namespace, backend_indices)
+    return ParsedExternalYaml(backend_key, autograd_key, class_name, cpp_namespace, backend_indices)
```

Comment:
Oh @guoyejun this breaks flake8 lint (see the CI failures in the PR)

Reply:
sure, will update, thanks.